### ISSUE TITLE
[gradle support] gradle plugin not found at jcenter

### ISSUE THREAD

**Issue by koppor** [Abnaxos/pegdown-doclet#43]
I'm trying to follow the process in [README.md of the gradle-plugin](https://github.com/Abnaxos/pegdown-doclet/blob/master/integrations/gradle-plugin/README.md). `gradlew tasks` gives me following output in my project:

```
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find ch.raffael.pegdown-doclet:gradle-plugin:1.0.
     Searched in the following locations:
         https://jcenter.bintray.com/ch/raffael/pegdown-doclet/gradle-plugin/1.0/gradle-plugin-1.0.pom
         https://jcenter.bintray.com/ch/raffael/pegdown-doclet/gradle-plugin/1.0/gradle-plugin-1.0.jar
         https://plugins.gradle.org/m2/ch/raffael/pegdown-doclet/gradle-plugin/1.0/gradle-plugin-1.0.pom
         https://plugins.gradle.org/m2/ch/raffael/pegdown-doclet/gradle-plugin/1.0/gradle-plugin-1.0.jar
```

When browsing https://jcenter.bintray.com/ch/raffael/pegdown-doclet/, only `pegdown-doclet/` appears.

**Comment by Abnaxos** [comment]
Oops. I'll release 1.2 ASAP.

For now, use the following:

```groovy
buildscript {
    repositories {
        maven {
            // switch to sonatype SNAPSHOT repository
            url 'https://oss.sonatype.org/content/repositories/snapshots/'
        }
    }
    dependencies {
        // use version 1.2-SNAPSHOT
        classpath 'ch.raffael.pegdown-doclet:gradle-plugin:1.2-SNAPSHOT'
    }
}

apply plugin: 'ch.raffael.doclets.pegdown'
```

**Comment by Abnaxos** [comment]
Ooops again. The artifact's name is `pegdown-doclet-gradle-plugin`.

**Comment by Abnaxos** [comment]
Fixed, 1.2 released. See README: The gradle plugin is now integrated in the main doclet jar:

```groovy
buildscript {
    repositories {
        mavenCentral() // or jcenter()
    }
    dependencies {
        classpath 'ch.raffael.pegdown-doclet:pegdown-doclet:1.2'
    }
}

apply plugin: 'ch.raffael.pegdown-doclet'
```
